src/components/Message/Message.scss

```diff
@@ -1,3 +1,8 @@
+@mixin fonts {
+    font-size: .8125em;
+    line-height: 1.2em;
+}
```

Comment:
@Farhan-emgage use existing typography styles, not new mixins.

Reply:
@Farhan-emgage TODO: Let's get this resolved during our meeting tomorrow.